src/transformers/models/flaubert/tokenization_flaubert.py

```diff
@@ -260,11 +260,13 @@ def __init__(
             additional_special_tokens=additional_special_tokens,
             lang2id=lang2id,
             id2lang=id2lang,
-            do_lowercase_and_remove_accent=do_lowercase_and_remove_accent,
```

Comment:
Passing this made (just a bit) sense before #19330 when it was still a subclass of `XLMTokenizer`.
After that PR, we can just set it to `False` without passing to `super().__init__`.